packages/block-editor/src/components/iframe/content.scss

```diff
@@ -0,0 +1,44 @@
+.block-editor-iframe__html {
+	transform-origin: top center;
+	transition: transform 0.3s;
+	@include reduce-motion("transition");
+}
+
+.block-editor-iframe__html.is-zoomed-out {
+	$scale: var(--wp-zoom-out-scale);
+	$frame-size: var(--wp-zoom-out-frame-size);
```

Comment:
Why did you move everything to CSS? We only need `--wp-zoom-out-scale`?
This is going to be problematic when I try to animate both scroll and scale in JS at the same time with the same function.

Reply:
Or I wonder if animating these variables will be the same in terms of performance. But still, I don't get why we make all of these variables if it's not needed.

Reply:
We might also want to prefix it with `wp-block-editor`, not sure

Reply:
> Or I wonder if animating these variables will be the same in terms of performance. But still, I don't get why we make all of these variables if it's not needed.

It was easier to read and seemed similar for performance. 

> This is going to be problematic when I try to animate both scroll and scale in JS at the same time with the same function.

Why do you need to animate them in JS? Can't the animations be done with CSS?